int64 ped_id
geometry_msg/Point32 rob_pose
geometry_msg/Point32 ped_pose

Header header

int64 action
